src/theme/MuiThemeProvider/muiTheme.ts

```diff
@@ -124,6 +124,7 @@ export default createTheme({
     h4: {
       fontSize: '1.25rem',
       fontWeight: 600,
+      lineHeight: '1.875rem',
```

Comment:
This can be removed (since we now apply 1.5 as line height for all text elements).